doc/examples/segmentation/plot_metrics.py

```diff
@@ -0,0 +1,145 @@
+"""
+========================
+Evaluating segmentations
```

Comment:
Segmentation metrics (?)

Reply:
I changed to `Evaluating segmentation metrics`